### DISCUSSION TITLE
Wish GitHub Mobile could manage a local and remote repo on mobile just like Git Journal app

### DISCUSSION THREAD

**Discussion by MrCharteredCurator** [community/community#10711] [Mobile] [answered]
There is GitHub desktop. I understand GitHub Mobile was made for restricted purpose. But I think some capabilities of GitHub Mobile could be extended.

I use Git Journal, an app to store notes in GitHub repo. But it has its own limitations.

There is no alternative I found.

What if GitHub Mobile could be a full fledged git client on Android (atleast) that creates, pushes, pulls, and do other git control activities on local and remote repos…?

**Comment by amrehman** [comment, marked as answer]
Hi @MrCharteredCurator ! 
Thanks a lot for this feedback and the feature request. 

We will discuss this amongst our team.
Such a workflow is not on our radar for now for the mobile clients, but we will keep an eye out for similar requests from our users and it will help us prioritise. 

Please do keep the feedback coming.

Thanks a lot.
